cli/Cargo.toml

```diff
@@ -33,16 +33,13 @@ clap = { version = "4.4.2", default-features = false, features = [
     "env",
 ] }
 colored = { version = "2.0.4", default-features = false }
-dialoguer = { version = "0.10.4", default-features = false, features = [
-    "history",
-    "password",
-    "completion",
-] }
+dialoguer = { version = "0.10.4", default-features = false, features = [ "password" ] }
```

Comment:
Doesn't look formatted

Reply:
hmm, what would you expect?

Reply:
No spaces around the feature, like for rustyline. Cargo fmt is supposed to fix that.

Reply:
I don't think my cargo fmt does that

Reply:
Fixed